src/components/tables/tables.scss

```diff
@@ -1,152 +1,197 @@
 @import '../../assets/scss/variables';
 //// Table Elements
 
- table {
+.table__responsive-container {
   width: 100%;
-  font-size: $content-font-size;
-}
+  position: relative;
 
- table {
-  border-spacing: 0;
-  border-collapse: collapse;
-}
+  &.table__row-headers--true {
+    .table__responsive-header__track-heading:first-child {
+      left: 0;
+    }
 
- table td,
- table th {
-  padding: $md;
-  vertical-align: top;
-  padding: $md;
-}
+    &.table--left-sticky-minwidth .table__responsive-header__track-heading:first-child {
+      position: sticky;
+    }
+  }
 
- table th {
-  color: $secondary;
-}
+  &.table--striped {
+    &.table--is-sticky-left.table--left-sticky-minwidth {
+      table.is-striped tbody th {
+        filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.2));
+      }
 
- table th:not([align]) {
-  text-align: left;
-}
+      .table__responsive-header__track-heading:first-child {
+        filter: drop-shadow(0px 4px 2px rgba(0, 0, 0, 0.2));
+      }
+    }
+  }
 
- table thead td,
- table thead th {
-  color: $grey;
-  border-width: 0 0 2px;
-}
+  &.table--is-sticky-left.table--left-sticky-minwidth .table__responsive-header__track-heading:first-child {
+    border-right: 1px solid #ccc;
+  }
 
- table tfoot td,
- table tfoot th {
-  color: $grey;
-  border-width: 2px 0 0;
-}
+  .table__responsive-header__track-heading:first-child {
+    border-right: 1px solid #fff;
+    transition: border-right 0.3s ease-in-out;
+  }
 
- table tbody tr:last-child td,
- table tbody tr:last-child th {
-  border-bottom-width: 0;
+  &.table--is-sticky-left.table--striped.table--left-sticky-minwidth .table__responsive-header__track-heading:first-child {
+    border-right: none;
+  }
+
+  &.table--left-sticky-minwidth table tbody th {
+    position: sticky;
+  }
 }
 
 table {
-  color: $secondary;
-  background-color: $white;
+  width: 100%;
+  font-size: $content-font-size;
+  border-spacing: 0;
 }
 
-table td,
-table th {
-  //vertical-align: top;
-  //border: 1px solid $light;
-  //border-width: 0 0 1px;
- // @include padding($top: $xsm, $right: $sm, $bottom: $xsm, $left: $sm);
-}
+.table__responsive-header {
+  position: sticky;
+  z-index: 1;
+  top: 0;
 
-table th {
-  color: $secondary;
-}
+  .table__responsive-header__scroller {
+    white-space: nowrap;
+    overflow-x: hidden;
 
-table th:not([align]) {
-  text-align: left;
+    .table__responsive-header__track-heading {
```

Comment:
Is it possible to add a brief comment on what the `track-heading` is?  This is the only class that I wasn't sure about after reading through this.

Reply:
So actually i was debating on this one to make it larger and or more descriptive, however, i thought `.table__responsive-header__scroll-track__track-heading` felt really long? it doe add clarity but I was unsure of whether that was worth it or not. Should i change the class?

Reply:
I think he means add to the comments.

Reply:
@GaryRidgway I think this is actually too verbose. I think something like `.table__sticky-heading` is sufficient.

Reply:
Right, I was trying to start a conversation on the core of the issue. I was wondering if BEM was explaining it in the right way or not, and if a comment was sufficient, or if it was better to rearrange the BEM class syntax. I can surely just add the comment, but I wanted to ask this first to try and solve the core of the issue instead of just put a band-aid on it.

Reply:
@pyrello  sounds good!

Reply:
@GaryRidgway Thanks for clarifying. I think that the BEM structure helps with readability, but comments are also necessary. Good discussion :)